server/sonar-web/src/main/js/apps/system/utils.ts

```diff
@@ -110,9 +110,17 @@ export function getNodeName(nodeInfo: NodeInfo): string {
   return nodeInfo['Name'];
 }
 
+function getSystemData(sysInfoData: SysInfo): SysValueObject {
+  const statData: SysValueObject = {};
+  if (sysInfoData['Statistics']) {
+    statData['Lines of Code'] = (sysInfoData['Statistics'] as SysValueObject)['ncloc'];
```

Comment:
I wonder if it's easy to format this number.

Reply:
It is, at first I didn't want to because all other numbers won't be formatted